metric/metricexport/reader.go

```diff
@@ -172,6 +172,20 @@ func (ir *IntervalReader) Stop() {
 	ir.quit = nil
 }
 
+// Flush flushes the metrics if IntervalReader is stopped, otherwise no-op.
+// Additional call to Flush are no-ops.
```

Comment:
remove this comment, as it no longer applies

Reply:
done